.github/actions/release/run.sh

```diff
@@ -67,7 +67,10 @@ do
         html_url="https://example.com"
         step="step"
     else
-        job_id=$(curl --silent "https://api.github.com/repos/dfinity/internet-identity/actions/runs/$GITHUB_RUN_ID/jobs" \
+        # We have so many jobs in our CI run that we need to explicitly increase the number of jobs per page in order to
+        # not run into errors due to pagination (default is 30 jobs per page).
+        # See https://docs.github.com/en/rest/actions/workflow-jobs?apiVersion=2022-11-28#list-jobs-for-a-workflow-run
+        job_id=$(curl --silent "https://api.github.com/repos/dfinity/internet-identity/actions/runs/$GITHUB_RUN_ID/jobs?per_page=100" \
```

Comment:
probably best to paginate then wdyt?

Reply:
I feel like more than 100 jobs is excessive for a single CI pipeline. Also, I'd rather not add more complexity to that script.

So, alternative suggestion: stop linking to the GitHub actions run in the release notes.
It does not seem that useful anyway because:
* people outside of the DFINITY organization cannot see the workflow logs, thus cannot verify that GitHub actions actually produced that hash
* trusted neurons verify the hash on their local machines anyway

WDYT?

Reply:
Don't care too strongly! I think using `github-script` would probably be the right thing to do